src/metrics/nom.rs

```diff
@@ -68,6 +68,96 @@ impl Stats {
     }
 }
 
+#[inline(always)]
+fn is_child_identifier(node: &Node, id: u16) -> bool {
```

Comment:
Why having `identifier` in the function name: it seems unrelated to what the function is doing.